src/GitHubCodeReviewDashboard/GitHubCodeReviewDashboard/wwwroot/css/site.css

```diff
@@ -0,0 +1,11 @@
+﻿a {
+    text-decoration: none;
+}
+
+body {
+    font-family: sans-serif;
+}
+
+.category a {
+    font-size: small;
```

Comment:
why?

Reply:
This is the filter link in the header:

![image](https://user-images.githubusercontent.com/201340/81622696-94e34d00-93a6-11ea-8498-77346db8b565.png)